staker/stateless_block_validator.go

```diff
@@ -473,10 +304,7 @@ func (v *StatelessBlockValidator) ValidationEntryAddSeqMessage(ctx context.Conte
 			continue
 		}
 		if v.daService == nil {
-			log.Error("No DAS configured, but sequencer message found with DAS header")
-			if v.blockchain.Config().ArbitrumChainParams.DataAvailabilityCommittee {
-				return errors.New("processing data availability chain without DAS configured")
-			}
+			log.Warn("No DAS configured, but sequencer message found with DAS header")
```

Comment:
It's not necessarily a config error. It could easily be an input error: A DAS header for non-DAS chain is simply junk/bad/malicious/wrong batch. There is a correct way to validate it (like we can validate any random/bad input that produces empty blocks).
If there really is a misconfiguration - validation (amongst other things) will fail, and hopefully the warnings will guide you.